src/panel.js

```diff
@@ -100,13 +103,43 @@ const addFullWidthButton = (fragment, listenerClass) => {
   return button;
 };
 
+const addTooltip = (wrapper, stringId) => {
+  const div = document.createElement("div");
+  div["id"] = stringId;
```

Comment:
I'm assuming this is just making a <div id=[stringId]> ? I didn't know you could do this. Cool.

Reply:
Yup. We have to fabricate all the HTML we inject in the extension (Both here in the content script and also in the panel).